### PR TITLE
Unminify favorites.js

### PR DESCRIPTION
**Pull request by Abbe98**

### CODE REVIEW

File: src/controllers/favorites.js [hunk 1]
```diff
@@ -127,31 +127,31 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
-                    CollapseBoxSetItems: !1,
+                    CollapseBoxSetItems: false,
                     ExcludeLocationTypes: "Virtual",
-                    EnableTotalRecordCount: !1
+                    EnableTotalRecordCount: false
                 };
             options.Limit = 20;
             var userId = apiClient.getCurrentUserId();
-            return "MusicArtist" === section.types ? apiClient.getArtists(userId, options) : (options.IncludeItemTypes = section.types, apiClient.getItems(userId, options))
+            return "MusicArtist" === section.types ? apiClient.getArtists(userId, options) : (options.IncludeItemTypes = section.types, apiClient.getItems(userId, options));
```
Comment (JustAMan) [inline src/controllers/favorites.js]:
```suggestion
            if ("MusicArtist" === section.types) {
                return apiClient.getArtists(userId, options);
            } else {
                options.IncludeItemTypes = section.types;
                return apiClient.getItems(userId, options);
            }
```

File: src/controllers/favorites.js [hunk 2]
```diff
@@ -178,12 +178,12 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
                 leadingButtons: leadingButtons,
                 lines: lines
-            })
+            });
         }
     }
 
     function FavoritesTab(view, params) {
-        this.view = view, this.params = params, this.apiClient = connectionManager.currentApiClient(), this.sectionsContainer = view.querySelector(".sections"), createSections(this, this.sectionsContainer, this.apiClient)
+        this.view = view, this.params = params, this.apiClient = connectionManager.currentApiClient(), this.sectionsContainer = view.querySelector(".sections"), createSections(this, this.sectionsContainer, this.apiClient);
```
Comment (JustAMan) [inline src/controllers/favorites.js]:
```suggestion
        this.view = view;
        this.params = params;
        this.apiClient = connectionManager.currentApiClient();
        this.sectionsContainer = view.querySelector(".sections");
        createSections(this, this.sectionsContainer, this.apiClient);
```

File: src/controllers/favorites.js [hunk 3]
```diff
@@ -192,25 +192,25 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
+            section.showTitle || (sectionClass += " verticalSection-extrabottompadding"), html += '<div class="' + sectionClass + ' hide">', html += '<div class="sectionTitleContainer sectionTitleContainer-cards padded-left">', layoutManager.tv ? html += '<h2 class="sectionTitle sectionTitle-cards">' + globalize.translate(section.name) + "</h2>" : (html += '<a is="emby-linkbutton" href="' + getRouteUrl(section, apiClient.serverId()) + '" class="more button-flat button-flat-mini sectionTitleTextButton">', html += '<h2 class="sectionTitle sectionTitle-cards">', html += globalize.translate(section.name), html += "</h2>", html += '<i class="md-icon">&#xE5CC;</i>', html += "</a>"), html += "</div>", html += '<div is="emby-scroller" class="padded-top-focusscale padded-bottom-focusscale" data-mousewheel="false" data-centerfocus="true"><div is="emby-itemscontainer" class="itemsContainer scrollSlider focuscontainer-x padded-left padded-right" data-monitor="markfavorite"></div></div>', html += "</div>" ;
```
Comment (JustAMan) [inline src/controllers/favorites.js]:
```suggestion
            if (!section.showTitle) {
                sectionClass += " verticalSection-extrabottompadding";
            }
            html += '<div class="' + sectionClass + ' hide">';
            html += '<div class="sectionTitleContainer sectionTitleContainer-cards padded-left">';
            if (layoutManager.tv) {
                html += '<h2 class="sectionTitle sectionTitle-cards">' + globalize.translate(section.name) + "</h2>";
            } else {
                html += '<a is="emby-linkbutton" href="' + getRouteUrl(section, apiClient.serverId()) + '" class="more button-flat button-flat-mini sectionTitleTextButton">';
                html += '<h2 class="sectionTitle sectionTitle-cards">';
                html += globalize.translate(section.name);
                html += "</h2>";
                html += '<i class="md-icon">&#xE5CC;</i>';
                html += "</a>";
            }
            html += "</div>";
            html += '<div is="emby-scroller" class="padded-top-focusscale padded-bottom-focusscale" data-mousewheel="false" data-centerfocus="true"><div is="emby-itemscontainer" class="itemsContainer scrollSlider focuscontainer-x padded-left padded-right" data-monitor="markfavorite"></div></div>';
            html += "</div>" ;
```

File: src/controllers/favorites.js [hunk 4]
```diff
@@ -192,25 +192,25 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
         }
         elem.innerHTML = html;
         var elems = elem.querySelectorAll(".itemsContainer");
         for (i = 0, length = elems.length; i < length; i++) {
             var itemsContainer = elems[i];
-            itemsContainer.fetchData = getFetchDataFn(sections[i]).bind(instance), itemsContainer.getItemsHtml = getItemsHtmlFn(sections[i]).bind(instance), itemsContainer.parentContainer = dom.parentWithClass(itemsContainer, "verticalSection")
+            itemsContainer.fetchData = getFetchDataFn(sections[i]).bind(instance), itemsContainer.getItemsHtml = getItemsHtmlFn(sections[i]).bind(instance), itemsContainer.parentContainer = dom.parentWithClass(itemsContainer, "verticalSection");
```
Comment (JustAMan) [inline src/controllers/favorites.js]:
```suggestion
            itemsContainer.fetchData = getFetchDataFn(sections[i]).bind(instance);
            itemsContainer.getItemsHtml = getItemsHtmlFn(sections[i]).bind(instance);
            itemsContainer.parentContainer = dom.parentWithClass(itemsContainer, "verticalSection");
```

File: src/controllers/favorites.js [hunk 5]
```diff
@@ -192,25 +192,25 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
     return FavoritesTab.prototype.onResume = function(options) {
         for (var promises = (this.apiClient, []), view = this.view, elems = this.sectionsContainer.querySelectorAll(".itemsContainer"), i = 0, length = elems.length; i < length; i++) promises.push(elems[i].resume(options));
         Promise.all(promises).then(function() {
-            options.autoFocus && focusManager.autoFocus(view)
-        })
+            options.autoFocus && focusManager.autoFocus(view);
+        });
     }, FavoritesTab.prototype.onPause = function() {
-        for (var elems = this.sectionsContainer.querySelectorAll(".itemsContainer"), i = 0, length = elems.length; i < length; i++) elems[i].pause()
+        for (var elems = this.sectionsContainer.querySelectorAll(".itemsContainer"), i = 0, length = elems.length; i < length; i++) elems[i].pause();
```
Comment (JustAMan) [inline src/controllers/favorites.js]:
```suggestion
        var elems = this.sectionsContainer.querySelectorAll(".itemsContainer");
        for (var i = 0, length = elems.length; i < length; i++) {
            elems[i].pause();
        }
```

File: src/controllers/favorites.js [hunk 6]
```diff
@@ -192,25 +192,25 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
         elem.innerHTML = html;
         var elems = elem.querySelectorAll(".itemsContainer");
         for (i = 0, length = elems.length; i < length; i++) {
             var itemsContainer = elems[i];
-            itemsContainer.fetchData = getFetchDataFn(sections[i]).bind(instance), itemsContainer.getItemsHtml = getItemsHtmlFn(sections[i]).bind(instance), itemsContainer.parentContainer = dom.parentWithClass(itemsContainer, "verticalSection")
+            itemsContainer.fetchData = getFetchDataFn(sections[i]).bind(instance), itemsContainer.getItemsHtml = getItemsHtmlFn(sections[i]).bind(instance), itemsContainer.parentContainer = dom.parentWithClass(itemsContainer, "verticalSection");
         }
     }
     return FavoritesTab.prototype.onResume = function(options) {
         for (var promises = (this.apiClient, []), view = this.view, elems = this.sectionsContainer.querySelectorAll(".itemsContainer"), i = 0, length = elems.length; i < length; i++) promises.push(elems[i].resume(options));
```
Comment (JustAMan) [inline src/controllers/favorites.js]:
```suggestion
         var promises = (this.apiClient, []);
         var view = this.view;
         var elems = this.sectionsContainer.querySelectorAll(".itemsContainer");
         for (var i = 0, length = elems.length; i < length; i++) {
             promises.push(elems[i].resume(options));
         }
```

**Comment by grafixeyehero** [comment]
@JustAMan i diminify favorites.js
 on this PR https://github.com/jellyfin/jellyfin-web/pull/548

**Comment by Abbe98** [comment]
@grafixeyehero awesome, I will close this.

**Pull request closed without merging**